
Lmod is automatically replacing "scicomp-python-env/2025.1" with "mamba/2025.1".

Assuming unrestricted shared filesystem usage.
Building DAG of jobs...
Your conda installation is not configured to use strict channel priorities. This is however crucial for having robust and correct environments (for details, see https://conda-forge.org/docs/user/tipsandtricks.html). Please consider to configure strict priorities by executing 'conda config --set channel_priority strict'.
Using shell: /usr/bin/bash
Provided cores: 4
Rules claiming more threads will be scaled down.
Job stats:
job                       count
----------------------  -------
all                           1
clean_features                3
cluster                       3
extract_globem                4
rename_and_concatenate        2
signature                     6
total                        19

Select jobs to execute...
Execute 4 jobs...

[Thu Jun 26 17:12:59 2025]
localrule extract_globem:
    input: /m/cs/work/luongn1/globem/INS-W_1/FeatureData/sleep.csv, /m/cs/work/luongn1/globem/INS-W_2/FeatureData/sleep.csv, /m/cs/work/luongn1/globem/INS-W_3/FeatureData/sleep.csv, /m/cs/work/luongn1/globem/INS-W_4/FeatureData/sleep.csv, /m/cs/work/luongn1/globem/pid_mappings.csv
    output: data/interim/globem/sleep_4epochs.csv
    jobid: 16
    reason: Params have changed since last execution
    wildcards: sensor=sleep
    resources: tmpdir=/tmp

[Thu Jun 26 17:12:59 2025]
localrule rename_and_concatenate:
    input: data/interim/tesserae/screen_4epochs.csv, data/interim/tesserae/call_4epochs.csv, data/interim/tesserae/steps_4epochs.csv, data/interim/tesserae/sleep_4epochs.csv
    output: data/processed/tesserae/all_features.csv
    jobid: 3
    reason: Code has changed since last execution
    wildcards: study=tesserae
    resources: tmpdir=/tmp

[Thu Jun 26 17:12:59 2025]
localrule extract_globem:
    input: /m/cs/work/luongn1/globem/INS-W_1/FeatureData/screen.csv, /m/cs/work/luongn1/globem/INS-W_2/FeatureData/screen.csv, /m/cs/work/luongn1/globem/INS-W_3/FeatureData/screen.csv, /m/cs/work/luongn1/globem/INS-W_4/FeatureData/screen.csv, /m/cs/work/luongn1/globem/pid_mappings.csv
    output: data/interim/globem/screen_4epochs.csv
    jobid: 13
    reason: Params have changed since last execution
    wildcards: sensor=screen
    resources: tmpdir=/tmp

[Thu Jun 26 17:12:59 2025]
localrule extract_globem:
    input: /m/cs/work/luongn1/globem/INS-W_1/FeatureData/steps.csv, /m/cs/work/luongn1/globem/INS-W_2/FeatureData/steps.csv, /m/cs/work/luongn1/globem/INS-W_3/FeatureData/steps.csv, /m/cs/work/luongn1/globem/INS-W_4/FeatureData/steps.csv, /m/cs/work/luongn1/globem/pid_mappings.csv
    output: data/interim/globem/steps_4epochs.csv
    jobid: 15
    reason: Params have changed since last execution
    wildcards: sensor=steps
    resources: tmpdir=/tmp

Activating conda environment: .snakemake/conda/72e6074358eff52e7b6a2a521babd7e1_
Activating conda environment: .snakemake/conda/72e6074358eff52e7b6a2a521babd7e1_
Activating conda environment: .snakemake/conda/72e6074358eff52e7b6a2a521babd7e1_
Activating conda environment: .snakemake/conda/72e6074358eff52e7b6a2a521babd7e1_
Activating conda environment: .snakemake/conda/72e6074358eff52e7b6a2a521babd7e1_
Activating conda environment: .snakemake/conda/72e6074358eff52e7b6a2a521babd7e1_
Activating conda environment: .snakemake/conda/72e6074358eff52e7b6a2a521babd7e1_
Activating conda environment: .snakemake/conda/72e6074358eff52e7b6a2a521babd7e1_
/scratch/work/luongn1/digirhythm-v2/workflow/rules/../scripts/preprocess/globem/base.py:51: DtypeWarning: Columns (808,809,810,811,812,813,814) have mixed types. Specify dtype option on import or set low_memory=False.
  df = pd.read_csv(fn, index_col=0)
/scratch/work/luongn1/digirhythm-v2/workflow/rules/../scripts/preprocess/globem/base.py:51: DtypeWarning: Columns (808,809,810,811,812,813,814,820,834) have mixed types. Specify dtype option on import or set low_memory=False.
  df = pd.read_csv(fn, index_col=0)
/scratch/work/luongn1/digirhythm-v2/workflow/rules/../scripts/preprocess/globem/base.py:68: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame.
Try using .loc[row_indexer,col_indexer] = value instead

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  df[columns] = df[columns].fillna(0)
/scratch/work/luongn1/digirhythm-v2/workflow/rules/../scripts/preprocess/globem/base.py:77: FutureWarning: DataFrame.applymap has been deprecated. Use DataFrame.map instead.
  pid_mappings = pid_mappings.applymap(
Traceback (most recent call last):
  File "/scratch/work/luongn1/digirhythm-v2/.snakemake/scripts/tmpyptn4c4k.steps.py", line 64, in <module>
    main(snakemake.input, snakemake.output[0])
  File "/scratch/work/luongn1/digirhythm-v2/.snakemake/scripts/tmpyptn4c4k.steps.py", line 58, in main
    res = processor.re_id_returning_users(res)
  File "/scratch/work/luongn1/digirhythm-v2/workflow/rules/../scripts/preprocess/globem/base.py", line 82, in re_id_returning_users
    df["unique_id"] = df.apply(
  File "/scratch/work/luongn1/digirhythm-v2/.snakemake/conda/72e6074358eff52e7b6a2a521babd7e1_/lib/python3.9/site-packages/pandas/core/frame.py", line 10374, in apply
    return op.apply().__finalize__(self, method="apply")
  File "/scratch/work/luongn1/digirhythm-v2/.snakemake/conda/72e6074358eff52e7b6a2a521babd7e1_/lib/python3.9/site-packages/pandas/core/apply.py", line 916, in apply
    return self.apply_standard()
  File "/scratch/work/luongn1/digirhythm-v2/.snakemake/conda/72e6074358eff52e7b6a2a521babd7e1_/lib/python3.9/site-packages/pandas/core/apply.py", line 1063, in apply_standard
    results, res_index = self.apply_series_generator()
  File "/scratch/work/luongn1/digirhythm-v2/.snakemake/conda/72e6074358eff52e7b6a2a521babd7e1_/lib/python3.9/site-packages/pandas/core/apply.py", line 1081, in apply_series_generator
    results[i] = self.func(v, *self.args, **self.kwargs)
  File "/scratch/work/luongn1/digirhythm-v2/workflow/rules/../scripts/preprocess/globem/base.py", line 83, in <lambda>
    lambda row: "".join(f"{int(val):d}" for val in row if pd.notnull(val)),
  File "/scratch/work/luongn1/digirhythm-v2/workflow/rules/../scripts/preprocess/globem/base.py", line 83, in <genexpr>
    lambda row: "".join(f"{int(val):d}" for val in row if pd.notnull(val)),
ValueError: invalid literal for int() with base 10: 'INS-W_001'
/scratch/work/luongn1/digirhythm-v2/workflow/rules/../scripts/preprocess/globem/base.py:108: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame.
Try using .loc[row_indexer,col_indexer] = value instead

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  df[f"{col}:within_norm"] = df.groupby(self.groupby_cols)[col].transform(
/scratch/work/luongn1/digirhythm-v2/workflow/rules/../scripts/preprocess/globem/base.py:51: DtypeWarning: Columns (52,53,54,55,56,57,58,59,60,61,62,63,64,65,66,67,68,69,70,71,72,73,74,75,76,77,78,79,178,179,180,181,182,183,184,185,186,187,188,189,190,191,192,193,194,195,196,197,198,199,200,201,202,203,204,205,304,305,306,307,308,309,310,311,312,313,314,315,316,317,318,319,320,321,322,323,324,325,326,327,328,329,330,331,430,431,432,433,434,435,436,437,438,439,440,441,442,443,444,445,446,447,448,449,450,451,452,453,454,455,456,457,556,557,558,559,560,561,562,563,564,565,566,567,568,569,570,571,572,573,574,575,576,577,578,579,580,581,582,583,682,683,684,685,686,687,688,689,690,691,692,693,694,695,696,697,698,699,700,701,702,703,704,705,706,707,708,709,808,809,810,811,812,813,814,815,816,817,818,819,820,821,822,823,824,825,826,827,828,829,830,831,832,833,834,835,934,935,936,937,938,939,940,941,942,943,944,945,946,947,948,949,950,951,952,953,954,955,956,957,958,959,960,961,1060,1061,1062,1063,1064,1065,1066,1067,1068,1069,1070,1071,1072,1073,1074,1075,1076,1077,1078,1079,1080,1081,1082,1083,1084,1085,1086,1087) have mixed types. Specify dtype option on import or set low_memory=False.
  df = pd.read_csv(fn, index_col=0)
/scratch/work/luongn1/digirhythm-v2/workflow/rules/../scripts/preprocess/globem/base.py:108: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame.
Try using .loc[row_indexer,col_indexer] = value instead

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  df[f"{col}:within_norm"] = df.groupby(self.groupby_cols)[col].transform(
/scratch/work/luongn1/digirhythm-v2/workflow/rules/../scripts/preprocess/globem/base.py:77: FutureWarning: DataFrame.applymap has been deprecated. Use DataFrame.map instead.
  pid_mappings = pid_mappings.applymap(
Traceback (most recent call last):
  File "/scratch/work/luongn1/digirhythm-v2/.snakemake/scripts/tmpuycq99hz.sleep.py", line 83, in <module>
    main(snakemake.input, snakemake.output[0])
  File "/scratch/work/luongn1/digirhythm-v2/.snakemake/scripts/tmpuycq99hz.sleep.py", line 77, in main
    res = processor.re_id_returning_users(res)
  File "/scratch/work/luongn1/digirhythm-v2/workflow/rules/../scripts/preprocess/globem/base.py", line 82, in re_id_returning_users
    df["unique_id"] = df.apply(
  File "/scratch/work/luongn1/digirhythm-v2/.snakemake/conda/72e6074358eff52e7b6a2a521babd7e1_/lib/python3.9/site-packages/pandas/core/frame.py", line 10374, in apply
    return op.apply().__finalize__(self, method="apply")
  File "/scratch/work/luongn1/digirhythm-v2/.snakemake/conda/72e6074358eff52e7b6a2a521babd7e1_/lib/python3.9/site-packages/pandas/core/apply.py", line 916, in apply
    return self.apply_standard()
  File "/scratch/work/luongn1/digirhythm-v2/.snakemake/conda/72e6074358eff52e7b6a2a521babd7e1_/lib/python3.9/site-packages/pandas/core/apply.py", line 1063, in apply_standard
    results, res_index = self.apply_series_generator()
  File "/scratch/work/luongn1/digirhythm-v2/.snakemake/conda/72e6074358eff52e7b6a2a521babd7e1_/lib/python3.9/site-packages/pandas/core/apply.py", line 1081, in apply_series_generator
    results[i] = self.func(v, *self.args, **self.kwargs)
  File "/scratch/work/luongn1/digirhythm-v2/workflow/rules/../scripts/preprocess/globem/base.py", line 83, in <lambda>
    lambda row: "".join(f"{int(val):d}" for val in row if pd.notnull(val)),
  File "/scratch/work/luongn1/digirhythm-v2/workflow/rules/../scripts/preprocess/globem/base.py", line 83, in <genexpr>
    lambda row: "".join(f"{int(val):d}" for val in row if pd.notnull(val)),
ValueError: invalid literal for int() with base 10: 'INS-W_001'
RuleException:
CalledProcessError in file /scratch/work/luongn1/digirhythm-v2/workflow/rules/preprocess.smk, line 57:
Command 'source /appl/scibuilder-mamba/aalto-rhel9/prod/software/mamba/2025.1/f67be15/bin/activate '/scratch/work/luongn1/digirhythm-v2/.snakemake/conda/72e6074358eff52e7b6a2a521babd7e1_'; set -euo pipefail;  python /scratch/work/luongn1/digirhythm-v2/.snakemake/scripts/tmpyptn4c4k.steps.py' returned non-zero exit status 1.
[Thu Jun 26 17:13:30 2025]
Error in rule extract_globem:
    jobid: 15
    input: /m/cs/work/luongn1/globem/INS-W_1/FeatureData/steps.csv, /m/cs/work/luongn1/globem/INS-W_2/FeatureData/steps.csv, /m/cs/work/luongn1/globem/INS-W_3/FeatureData/steps.csv, /m/cs/work/luongn1/globem/INS-W_4/FeatureData/steps.csv, /m/cs/work/luongn1/globem/pid_mappings.csv
    output: data/interim/globem/steps_4epochs.csv
    conda-env: /scratch/work/luongn1/digirhythm-v2/.snakemake/conda/72e6074358eff52e7b6a2a521babd7e1_

RuleException:
CalledProcessError in file /scratch/work/luongn1/digirhythm-v2/workflow/rules/preprocess.smk, line 57:
Command 'source /appl/scibuilder-mamba/aalto-rhel9/prod/software/mamba/2025.1/f67be15/bin/activate '/scratch/work/luongn1/digirhythm-v2/.snakemake/conda/72e6074358eff52e7b6a2a521babd7e1_'; set -euo pipefail;  python /scratch/work/luongn1/digirhythm-v2/.snakemake/scripts/tmpuycq99hz.sleep.py' returned non-zero exit status 1.
[Thu Jun 26 17:13:30 2025]
Error in rule extract_globem:
    jobid: 16
    input: /m/cs/work/luongn1/globem/INS-W_1/FeatureData/sleep.csv, /m/cs/work/luongn1/globem/INS-W_2/FeatureData/sleep.csv, /m/cs/work/luongn1/globem/INS-W_3/FeatureData/sleep.csv, /m/cs/work/luongn1/globem/INS-W_4/FeatureData/sleep.csv, /m/cs/work/luongn1/globem/pid_mappings.csv
    output: data/interim/globem/sleep_4epochs.csv
    conda-env: /scratch/work/luongn1/digirhythm-v2/.snakemake/conda/72e6074358eff52e7b6a2a521babd7e1_

/scratch/work/luongn1/digirhythm-v2/workflow/rules/../scripts/preprocess/globem/base.py:51: DtypeWarning: Columns (71,78,197,204,304,305,306,307,308,309,310,311,312,313,314,315,316,317,318,319,320,321,322,323,324,325,326,327,328,329,330,331,430,431,432,433,434,435,436,437,438,439,440,441,442,443,444,445,446,447,448,449,450,451,452,453,454,455,456,457,556,557,558,559,560,561,562,563,564,565,566,567,568,569,570,571,572,573,574,575,576,577,578,579,580,581,582,583,682,683,684,685,686,687,688,689,690,691,692,693,694,695,696,697,698,699,700,701,702,703,704,705,706,707,708,709,808,809,810,811,812,813,814,815,816,817,818,819,820,821,822,823,824,825,826,827,828,829,830,831,832,833,834,835,934,935,936,937,938,939,940,941,942,943,944,945,946,947,948,949,950,951,952,953,954,955,956,957,958,959,960,961,1060,1061,1062,1063,1064,1065,1066,1067,1068,1069,1070,1071,1072,1073,1074,1075,1076,1077,1078,1079,1080,1081,1082,1083,1084,1085,1086,1087) have mixed types. Specify dtype option on import or set low_memory=False.
  df = pd.read_csv(fn, index_col=0)
/scratch/work/luongn1/digirhythm-v2/workflow/rules/../scripts/preprocess/globem/base.py:108: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame.
Try using .loc[row_indexer,col_indexer] = value instead

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  df[f"{col}:within_norm"] = df.groupby(self.groupby_cols)[col].transform(
/scratch/work/luongn1/digirhythm-v2/workflow/rules/../scripts/preprocess/globem/base.py:108: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame.
Try using .loc[row_indexer,col_indexer] = value instead

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  df[f"{col}:within_norm"] = df.groupby(self.groupby_cols)[col].transform(
/scratch/work/luongn1/digirhythm-v2/workflow/rules/../scripts/preprocess/globem/base.py:108: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame.
Try using .loc[row_indexer,col_indexer] = value instead

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  df[f"{col}:within_norm"] = df.groupby(self.groupby_cols)[col].transform(
/scratch/work/luongn1/digirhythm-v2/workflow/rules/../scripts/preprocess/globem/base.py:108: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame.
Try using .loc[row_indexer,col_indexer] = value instead

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  df[f"{col}:within_norm"] = df.groupby(self.groupby_cols)[col].transform(
/scratch/work/luongn1/digirhythm-v2/workflow/rules/../scripts/preprocess/globem/base.py:108: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame.
Try using .loc[row_indexer,col_indexer] = value instead

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  df[f"{col}:within_norm"] = df.groupby(self.groupby_cols)[col].transform(
/scratch/work/luongn1/digirhythm-v2/.snakemake/scripts/tmp274oucxy.screen.py:48: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame.
Try using .loc[row_indexer,col_indexer] = value instead

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  df[columns] = df[columns].fillna(0)
/scratch/work/luongn1/digirhythm-v2/workflow/rules/../scripts/preprocess/globem/base.py:77: FutureWarning: DataFrame.applymap has been deprecated. Use DataFrame.map instead.
  pid_mappings = pid_mappings.applymap(
Traceback (most recent call last):
  File "/scratch/work/luongn1/digirhythm-v2/.snakemake/scripts/tmp274oucxy.screen.py", line 78, in <module>
    main(snakemake.input, snakemake.output[0])
  File "/scratch/work/luongn1/digirhythm-v2/.snakemake/scripts/tmp274oucxy.screen.py", line 72, in main
    res = processor.re_id_returning_users(res)
  File "/scratch/work/luongn1/digirhythm-v2/workflow/rules/../scripts/preprocess/globem/base.py", line 82, in re_id_returning_users
    df["unique_id"] = df.apply(
  File "/scratch/work/luongn1/digirhythm-v2/.snakemake/conda/72e6074358eff52e7b6a2a521babd7e1_/lib/python3.9/site-packages/pandas/core/frame.py", line 10374, in apply
    return op.apply().__finalize__(self, method="apply")
  File "/scratch/work/luongn1/digirhythm-v2/.snakemake/conda/72e6074358eff52e7b6a2a521babd7e1_/lib/python3.9/site-packages/pandas/core/apply.py", line 916, in apply
    return self.apply_standard()
  File "/scratch/work/luongn1/digirhythm-v2/.snakemake/conda/72e6074358eff52e7b6a2a521babd7e1_/lib/python3.9/site-packages/pandas/core/apply.py", line 1063, in apply_standard
    results, res_index = self.apply_series_generator()
  File "/scratch/work/luongn1/digirhythm-v2/.snakemake/conda/72e6074358eff52e7b6a2a521babd7e1_/lib/python3.9/site-packages/pandas/core/apply.py", line 1081, in apply_series_generator
    results[i] = self.func(v, *self.args, **self.kwargs)
  File "/scratch/work/luongn1/digirhythm-v2/workflow/rules/../scripts/preprocess/globem/base.py", line 83, in <lambda>
    lambda row: "".join(f"{int(val):d}" for val in row if pd.notnull(val)),
  File "/scratch/work/luongn1/digirhythm-v2/workflow/rules/../scripts/preprocess/globem/base.py", line 83, in <genexpr>
    lambda row: "".join(f"{int(val):d}" for val in row if pd.notnull(val)),
ValueError: invalid literal for int() with base 10: 'INS-W_001'
RuleException:
CalledProcessError in file /scratch/work/luongn1/digirhythm-v2/workflow/rules/preprocess.smk, line 57:
Command 'source /appl/scibuilder-mamba/aalto-rhel9/prod/software/mamba/2025.1/f67be15/bin/activate '/scratch/work/luongn1/digirhythm-v2/.snakemake/conda/72e6074358eff52e7b6a2a521babd7e1_'; set -euo pipefail;  python /scratch/work/luongn1/digirhythm-v2/.snakemake/scripts/tmp274oucxy.screen.py' returned non-zero exit status 1.
[Thu Jun 26 17:13:33 2025]
Error in rule extract_globem:
    jobid: 13
    input: /m/cs/work/luongn1/globem/INS-W_1/FeatureData/screen.csv, /m/cs/work/luongn1/globem/INS-W_2/FeatureData/screen.csv, /m/cs/work/luongn1/globem/INS-W_3/FeatureData/screen.csv, /m/cs/work/luongn1/globem/INS-W_4/FeatureData/screen.csv, /m/cs/work/luongn1/globem/pid_mappings.csv
    output: data/interim/globem/screen_4epochs.csv
    conda-env: /scratch/work/luongn1/digirhythm-v2/.snakemake/conda/72e6074358eff52e7b6a2a521babd7e1_

[Thu Jun 26 17:13:40 2025]
Finished job 3.
1 of 19 steps (5%) done
Shutting down, this might take some time.
Exiting because a job execution failed. Look above for error message
Complete log: .snakemake/log/2025-06-26T171255.375907.snakemake.log
WorkflowError:
At least one job did not complete successfully.
